tests/pipelines/test_pipelines_text_generation.py

```diff
@@ -191,6 +189,68 @@ def test_small_chat_model_pt(self):
             ],
         )
 
+    @require_torch
+    def test_small_chat_model_continue_final_message(self):
+        # Here we check that passing a chat that ends in an assistant message is handled correctly
+        # by continuing the final message rather than starting a new one
+        text_generator = pipeline(
+            task="text-generation", model="rocketknight1/tiny-gpt2-with-chatml-template", framework="pt"
```

Comment:
Can you move that model to `hf-internal-testing`? We're trying to move away from user/contributor/maintainer-hosted checkpoints in our CI

Reply:
Done!